### PR TITLE
Mention admin guide in README

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by mschnitzer** [comment]
lgtm

File: README [hunk 1]
```diff
@@ -7,6 +7,7 @@ the Open Build Service (OBS).
 
 The documentation is currently split into two books:
```
Comment (eduardoj) [inline README]: two -> three. Isn't it?

Comment (bgeuken) [inline README]: Good catch

**Comment by bgeuken** [comment]
\*Updated\*

**Pull request merged**